releasenotes/notes/deprecate-algorithms-c6e1e28b6091c507.yaml

```diff
@@ -0,0 +1,39 @@
+---
+deprecations:
+  - |
+    All of the following features are now deprecated, after having been made pending deprecation
+    since 0.22.0. More information is available in the release notes of that version for the
+    replacements:
```

Comment:
Would it be better now to link to the migration guide, sure there was some info in the prior release notes for 0.22, but the guide is now much more comprehensive